### ISSUE TITLE
Writing partitions freezes GUI on MT6789

### ISSUE THREAD

**Issue by 4aiman** [bkerler/mtkclient#1239]
I'm using the latest mtkclient on windows 11 (git pull'ed on 10/05/24).
Was able to read  the partitions from a tablet,
But attempting to restore those (immediately after backing up, to test whether the backup is worth keeping at all) freezes after several kb or mb of data transferred.

E.g. 
I "selected from folder" everything and the process copied ``para` and froze on `misc` after just 1Mb.
I rebooted, "selected from folder" everything, removed `para` and `misc`  and the process froze on `userdata` after about 30Mb.
![](https://github.com/user-attachments/assets/a731f688-3316-417f-a8b6-d13d2b090000)

**Comment by 4aiman** [comment]
Worst part is that gui is unusable after that.
After a while it "enables" the `write partition(s)` button, but the progress bars aren't moving, status is showing `done`
![](https://github.com/user-attachments/assets/e6391a76-38e0-4d0b-b1f9-d7e9157ebda5)

**Comment by IsHacker003** [comment]
Yes the GUI is kinda buggy in this regard. Use the command line instead.

**Comment by 4aiman** [comment]
I did use the commandline and flashed via fastboot/fastbootd what I couldn't do with MTK, but that's kinda the whole point of the issue :D
